api/USBIsochronousInTransferPacket.json

```diff
@@ -0,0 +1,196 @@
+{
+  "api": {
+    "USBIsochronousInTransferPacket": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/USBIsochronousInTransferPacket",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "61"
+          },
+          "chrome_android": {
+            "version_added": "61"
+          },
+          "edge": {
+            "version_added": false
+          },
+          "edge_mobile": {
+            "version_added": false
+          },
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "48"
+          },
+          "opera_android": {
+            "version_added": "48"
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "USBIsochronousInTransferPacket": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/USBIsochronousInTransferPacket/USBIsochronousInTransferPacket",
```

Comment:
Please add a description clarifying that this is the constructor.
 `"description": "<code>USBIsochronousInTransferPacket()</code> constructor",`